output/adapter/result.go

```diff
@@ -0,0 +1,4 @@
+package adapter
+
+// Result is the result of the output adapter
+type Result interface{}
```

Comment:
maybe just return a string is enought currently

Reply:
Using struct is future-proofing